### DISCUSSION TITLE
3060 here.

### DISCUSSION THREAD

**Discussion by AbyszOne** [smy20011/dreambooth-gui#31] [Q&A]
As in old versions, i am able to make all the process, 800 steps, name, class, etc. But the ckpt didn't look any near or remotely close to my training images. Ive used same set in shivas colab with clean results. Any idea what i could be done wrong?

**Comment by futurevessel** [comment]
I've had successful training with dreambooth-gui, I'm also on a 3060.

In my experience, especially if you're training on faces (but probably overall), make sure you use --train_text_encoder as it improves the results greatly, however when enabling this I also need to enable --gradient_checkpointing (for VRAM reasons) which slows the training down. I think I've had good results with the following (copy paste into the 'training arguments' box):

--mixed_precision=fp16
--train_batch_size=1
--gradient_accumulation_steps=1
--use_8bit_adam
--resolution=512
--gradient_checkpointing
--train_text_encoder

Learning rate efficiency seems very tied to the number of steps you choose, but I primarily use 1e-6 and 2e-6, I think most collabs I've seen use 5e-6, good luck in experimenting :)

**Reply by AbyszOne** [reply]
What ive learned so far. 
1e5 works well without text-enc. 1000s or below get acceptable results. Just a bit rigid. Text-enc overtrain it. Avoid it.
1e6 works better with text-enc. At ~1500s get flexible and decent results. This is clearly a better path.
Dunno about different class images influence. And im using "person" for any character.

**Comment by smy20011** [comment]
Make sure you use newest version for training. There is a bug in old version that prevent you from training in 30-series card and It's fixed in v0.1.6

**Reply by AbyszOne** [reply]
Im on 1.8. It pick my images. I set all normal. Steps run. Ckpt is released. But... still ZERO training hint. Nothing is come from my keyword or Key + class, after half dozen variations. Just generic random stuff. 😥Its so frustrating tbh.

**Reply by smy20011** [reply]
Just found a bug that prevent you from fetching latest image. Do you mind delete the smy20011:dreambooth image and rerun again?

**Reply by smy20011** [reply]
Run `docker pull smy20011/dreambooth:latest` in cmd should do the trick.

**Reply by AbyszOne** [reply]
Well, it finally worked, after several random errors. FYI: 800s, the result is "not bad", in colab "good". With --gradient_checkpointing --train_text_encoder the result was "bad". Overtrained and deformed. (Anime character). All in 1e-5. Tips?

**Reply by futurevessel** [reply]
Well, it's not as if the colab has some 'secret sauce', you can look at the settings the colab uses and make sure you are using the same when running dreambooth-gui. Most interesting parameters would be the number of steps and the 'learning rate', is the colab using 1e-5 as the learning rate ?
